snapshots/README.md

```diff
@@ -39,24 +39,10 @@ plugins {
   id("com.google.devtools.ksp")
 }
 
-android {
```

Comment:
This is no longer necessary as of KSP 1.0.9: https://kotlinlang.org/docs/ksp-quickstart.html#make-ide-aware-of-generated-code